update-source-git/README.md

```diff
@@ -0,0 +1,105 @@
+`packit source-git update-source-git`
+
+This research covers a command which synchronizes changes from dist-git back to
+a source-git repo.
+
+Areas to cover:
+
+- spec file synchronization
+
+- sources synchronization
+
+- `.distro/source-git.yaml` update
+
+- code synchronization
+
+Let's break these down.
+
+## Spec file sync
+
+This seems trivial, we can copy the file from dist-git back to source-git. But
+before we do that, we should capture the version field and check if it differs
+from the source-git spec.
+
+## Sources synchronization
+
+Not much to do here, except that we should detect a removal of a file and also
+remove it in source-git's `.distro` folder.
+
+## `.distro/source-git.yaml` update
+
+Unless the version changes, there should be nothing to update. Actually this
```

Comment:
What should happen to this file if the version changes?

Reply:
I just realized we don't put source-git.yaml into the dist-git repo :man_facepalming:

Reply:
I added multiple clarifications about the version change in multiple sections